packages/mermaid/src/docs/syntax/flowchart.md

```diff
@@ -495,10 +502,10 @@ Beginner's tip—a full example using interactive links in a html context:
   </pre>
 
   <script>
-    const callback = function () {
+    var callback = function () {
       alert('A callback was triggered');
     };
-    const config = {
+    var config = {
       startOnLoad: true,
```

Comment:
These seem to still be var.

Reply:
probably changed in the wrong place.
 done